### DISCUSSION TITLE
Multizone heating Controller

### DISCUSSION THREAD

**Discussion by bsinfo523** [Beormund/Tasmota32-Multi-Zone-Heating-Controller#5] [General]
Hi Beormund,

I like the look of your berry coded multizone heating Controller.
Tasmota has a nice thermostat function with pi regulation of the temperature.

How is your multizone heating controller doing the regulation?

**Comment by Beormund** [comment]
The primary focus of the project was to replace my Horstmann H37XL three channel heating and hot water programmer with a Tasmota Berry scheduler. A couple of users asked if it would be possible to add thermostat input which I implemented at only a very basic level. For example, basic hysteresis needs to be added. The Tasmota thermostat feature is not included in precompiled binaries unfortunately. Frankly the codebase has gotten so big I could do with some help!

**Reply by Mike66912** [reply]
Hi Beormund
actual I did some tests with [Tasmota32-Multi-Zone-Heating-Controller](https://github.com/Beormund/Tasmota32-Multi-Zone-Heating-Controller) and I need for further activities a hysteresis. Where is this to added? 
like this:  ((Target+0,5grdC) - room) <0 power off; ((Target-0,5grdC)-room>0 power on 
Hysteresis 1grdC
Can You give me a little help?
Thanks Mike

**Comment by roesl** [comment]
Hi, how to load the software to tasmota module ?

**Reply by Beormund** [reply]
Via the Tasmota ESP32 UFS: https://tasmota.github.io/docs/UFS/
